sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/BUILD

```diff
@@ -277,6 +272,8 @@ opentitan_binary(
         "//sw/device/silicon_creator/lib:attestation_key_diversifiers",
         "//sw/device/silicon_creator/lib:error",
         "//sw/device/silicon_creator/lib:otbn_boot_services",
+        "//sw/device/silicon_creator/lib/cert:cdi_0_template_library",
```

Comment:
I think the cdi_0 is used in the code yet?

Reply:
fixed.